<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoronaSimulationDialog</class>
 <widget class="QDialog" name="CoronaSimulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="figuresLayout"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="estimateRButton">
     <property name="text">
      <string>Estimate R</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="actualDataFileButton">
       <property name="text">
        <string>Actual Data File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RDataFileButton">
       <property name="text">
        <string>R Data File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectivityDataFileButton">
       <property name="text">
        <string>Connectivity Data File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEditSimulationDays">
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Simulation Days</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actualDataFileButton</sender>
   <signal>clicked()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>actual_data_file_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>estimateRButton</sender>
   <signal>clicked()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>estimate_R()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSimulationDays</sender>
   <signal>textEdited(QString)</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>simulationDaysChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RDataFileButton</sender>
   <signal>clicked()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>r_data_file_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectivityDataFileButton</sender>
   <signal>clicked()</signal>
   <receiver>CoronaSimulationDialog</receiver>
   <slot>connectivity_data_file_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
